src/UpdateEventPage/UpdateEvent.js

```diff
@@ -19,6 +19,8 @@ import {
 } from "material-ui";
 
 import "../CreateEventsPage/css/create-event.css";
+import { userInfo } from "os";
```

Comment:
I don't think this is the `userInfo` you want. This one has to do with the operating system user (thus the `os` package) that's running the script, whereas you want the web session user. You'll probably want to pass this in from the enclosing component.